tests/Feature/Projects/TasksControllerTest.php

```
@@ -287,10 +287,10 @@ public function it_handles_special_characters_in_task_name(): void
 
         /* Act */
         $this->actingAs($user);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$user'. (undefined)

(UnusedLocalVariable)